.github/workflows/elixir.yml

```diff
@@ -0,0 +1,70 @@
+name: Elixir
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+    types: [opened, reopened, synchronize, labeled]
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  # test-matrix:
+  #   runs-on: ubuntu-latest
+  #   outputs:
+  #     matrix: ${{ steps.set-matrix.outputs.matrix }}
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - name: Read file
+  #       id: set-matrix
+  #       run: |
+  #         echo "::set-output name=matrix::$(cat ./.github/test-matrix.json | jq -r -s '.|tojson')"
+  opentelemetry-phoenix:
+    # needs: [test-matrix]
+    if: (contains(github.event.pull_request.labels.*.name, 'elixir') && contains(github.event.pull_request.labels.*.name, 'opentelemetry_phoenix'))
+    defaults:
+      run:
+        working-directory: instrumentation/${{ env.app }}
+    runs-on: ubuntu-18.04
+    name: Opentelemetry Phoenix test on Elixir ${{ matrix.elixir_version }} (OTP ${{ matrix.otp_version }})
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04]
+        otp_version: ["24.0.4", "23.3.4.2", "22.3.4.20"]
+        elixir_version: ["1.12.2", "1.11.4"]
```

Comment:
should also 1.10 be included? I remember from the opentelemetry_tesla library that 1.10 was mentioned as the lowest version in the OTEL land.

Probably unimportant, but I think patch version doesn't need to be included, e.g. `elixir_version: ["1.12", "1.11", "1.10"]`

Reply:
1.10 has its own include in the matrix. It's not compatible with OTP 24.